packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -509,12 +510,32 @@ abstract class IosAssetBundle extends Target {
         .copySync(dsymOutputBinary.path);
     }
 
+    final FlutterProject flutterProject = FlutterProject.fromDirectory(environment.projectDir);
+
+    bool isImpellerEnabled() {
+      final PlistParser plistParser = PlistParser(fileSystem: globals.fs, logger: environment.logger, processManager: globals.processManager);
```

Comment:
Nit, if you're using the cursed `globals`, might as well use `globals. plistParser`

Reply:
Ah, done.